### ISSUE TITLE
Unable to verify Domain

### ISSUE THREAD

**Issue by rushi0n** [is-a-dev/register#15007]
### I followed the steps yet I can't verify the domain
![Screenshot_20240720-185101](https://github.com/user-attachments/assets/ed6be9e2-6dfa-4aa8-9f7a-b45fe8b246a2)

**Comment by CuteDog5695** [comment]
Did your PR get merged or did you just make the file in your fork?

**Comment by rushi0n** [comment]
its merged

**Comment by CuteDog5695** [comment]
You made rushi.is-a.dev, not rushi0n.is-a.dev

**Comment by rushi0n** [comment]
oh so need to add rushi.is-a.dev as domain right

**Comment by CuteDog5695** [comment]
> oh so need to add rushi.is-a.dev as domain right

Yes, also I noticed that you made the github txt verification on rushi0n.is-a.dev, which we shouldn't of merged it. Do you want to rename that file to be on rushi.is-a.dev?

**Comment by rushi0n** [comment]
yes

**Comment by CuteDog5695** [comment]
Done, though next time please be careful about the file names.

**Comment by rushi0n** [comment]
okay

**Comment by CuteDog5695** [comment]
Is issue resolved now?
